acct-group/headscale/headscale-0.ebuild

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-group
+
+ACCT_GROUP_ID=600
```

Comment:
Use https://github.com/gentoo/api-gentoo-org/blob/master/bin/used_free_uidgids.sh script to determine the optimal UID/GID, it's different than this one ;-)

Reply:
yes, this should be done before we merge this PR

Reply:
ok, I'm going to mention pr now. After the pr is merged, I will update the corresponding commit here. I use this script to generate a new uid 514, and the corresponding commit will be updated later.

https://github.com/gentoo/api-gentoo-org/pull/514

Reply:
I've merged the api PR, can you update the UID/GID here?

Reply:
Yap, I am testing and will submit when the test passes

Reply:
> I've merged the api PR, can you update the UID/GID here?

I fixed the comment left, please review it again?

@thesamesam @dlan17